src/uproot/behaviors/TAxis.py

```diff
@@ -84,8 +83,7 @@ def __iter__(self):
             for x in fLabels:
                 yield str(x)
         else:
-            for low, high in self.intervals():
-                yield low, high
+            yield from self.intervals()
```

Comment:
:heart:

Reply:
That's clever! (Avoid unpacking and repacking that tuple, and the meaning is clearer, too. I haven't been using `yield from` for some reason that I don't remember, but it's certainly old enough to use it—it's not even a Python 3 thing.)

Reply:
It is very much a Python 3.3 thing. :) 3.3 happens to be over 9 years old, but it't newer than 2.7!

Reply:
Oh, I thought it was in Python 2. Well, if it started in Python 3.3 (9 years ago), then that's why I hadn't been using it, but can now.